python/pacemaker/_cts/environment.py

```diff
@@ -88,7 +88,8 @@ def dump(self):
 
         keys.sort()
         for key in keys:
-            self._logger.debug("Environment[%s]:\t%s" % (key, str(self[key])))
+            s = "Environment[%s]" % key
+            self._logger.debug("{key:35}: {val}".format(key=s, val=str(self[key])))
```

Comment:
can't use formatted parameters in 3.4, unfortunately

Reply:
This looks like it should be supported.  See 6.1.3.2 at https://docs.python.org/release/3.4.10/library/string.html#formatstrings, for instance.

Reply:
Ah cool, I didn't realize format() and formatted literals were different

Reply:
python has gone out of its way to make string formatting confusing.